src/annotator/util/scroll.js

```diff
@@ -69,3 +71,19 @@ export async function scrollElement(
     element.scrollTop = interpolate(startOffset, endOffset, scrollFraction);
   }
 }
+
+/**
+ * Smoothly scroll an element into view.
+ *
+ * @param {HTMLElement} element
+ * @param {object} options
+ *   @prop {number} maxDuration
```

Comment:
A little lingering confusion over `@param` vs `@prop`. Last note I saw from you was that `@prop` should only ever be used in `@typedef`s?

Reply:
Agh. This is a mistake. It should be `@param {number} [options.maxDuration]`.